web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -281,6 +281,7 @@ goog.require('gn_alert');
       },
       current: null,
       shibbolethEnabled: false,
+      shibbolethHideLogin: true,
```

Comment:
I maintained the shibbolethEnabled and created a new shibbolethHideLogin so the shibbolethEnabled can be used to redirect the login/logout button on a customization of the user interface.